<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RLTraining</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>640</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="listLabel">
         <property name="text">
          <string>Entrainement disponible</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="trainingListView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="removeTrainingBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Suprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addTrainingBtn">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Map disponible</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="mapListView">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="associateBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Associer</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>49</y>
      <width>781</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Configuration actuelle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="configurationListView"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RÃ©initialiser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer le jeux</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>trainingListView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>showMapList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapListView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>showAssociateBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>associateBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>associateTrainingToMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configurationListView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>showTrainingRemoveBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetTraining()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeTrainingConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTrainingBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAddTrainingWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeTrainingBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeTraining()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showFileDialog()</slot>
  <slot>showMapList()</slot>
  <slot>showAssociateBtn()</slot>
  <slot>associateTrainingToMap()</slot>
  <slot>showTrainingRemoveBtn()</slot>
  <slot>resetTraining()</slot>
  <slot>removeTrainingConfiguration()</slot>
  <slot>startGame()</slot>
  <slot>showAddTrainingWindow()</slot>
  <slot>removeTraining()</slot>
 </slots>
</ui>
